tests/btc/transactions.test.ts

```diff
@@ -805,6 +809,185 @@ describe('bitcoin transactions', () => {
     expect(signedTx.fee.toNumber()).eq(fee.toNumber());
   });
 
+  it('can calculate fee for ordinal send transaction', async () => {
+    const network = 'Mainnet';
```

Comment:
it seems like a lot of this const variables can be reused in this same file, right? maybe we can move them to a higher scope to DRY. or to a mock_data file